streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/Serialized.scala

```diff
@@ -18,7 +18,7 @@ package org.apache.kafka.streams.scala.kstream
 
 import org.apache.kafka.common.serialization.Serde
 import org.apache.kafka.streams.kstream.{Serialized => SerializedJ}
-
+@deprecated
 object Serialized {
```

Comment:
This is actually not referenced at all anymore, right? Maybe we should just delete it.

Reply:
see updated PR